[2025-09-03T14:53:13.138+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-03T14:53:13.163+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: foreign_1_live.save_foreign_1 scheduled__2025-09-02T23:46:00+00:00 [queued]>
[2025-09-03T14:53:13.171+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: foreign_1_live.save_foreign_1 scheduled__2025-09-02T23:46:00+00:00 [queued]>
[2025-09-03T14:53:13.171+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 11
[2025-09-03T14:53:13.183+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): save_foreign_1> on 2025-09-02 23:46:00+00:00
[2025-09-03T14:53:13.192+0000] {standard_task_runner.py:63} INFO - Started process 2764739 to run task
[2025-09-03T14:53:13.197+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'foreign_1_live', 'save_foreign_1', 'scheduled__2025-09-02T23:46:00+00:00', '--job-id', '74544', '--raw', '--subdir', 'DAGS_FOLDER/dag_foreign_1_live.py', '--cfg-path', '/tmp/tmp8bkzpvnz']
[2025-09-03T14:53:13.200+0000] {standard_task_runner.py:91} INFO - Job 74544: Subtask save_foreign_1
[2025-09-03T14:53:13.267+0000] {task_command.py:426} INFO - Running <TaskInstance: foreign_1_live.save_foreign_1 scheduled__2025-09-02T23:46:00+00:00 [running]> on host 1877907bc043
[2025-09-03T14:53:13.379+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='foreign_1_live' AIRFLOW_CTX_TASK_ID='save_foreign_1' AIRFLOW_CTX_EXECUTION_DATE='2025-09-02T23:46:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-02T23:46:00+00:00'
[2025-09-03T14:53:13.380+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-03T14:53:13.395+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-03T14:53:13.884+0000] {foreign_1.py:28} INFO - Đã lưu foreign_ALL
[2025-09-03T14:53:13.887+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-03T14:53:14.412+0000] {foreign_1.py:28} INFO - Đã lưu foreign_HOSE
[2025-09-03T14:53:14.413+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-03T14:53:15.180+0000] {foreign_1.py:28} INFO - Đã lưu foreign_HNX
[2025-09-03T14:53:15.182+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-03T14:53:15.675+0000] {foreign_1.py:28} INFO - Đã lưu foreign_UPCOM
[2025-09-03T14:53:15.676+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-09-03T14:53:15.677+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-03T14:53:15.688+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=foreign_1_live, task_id=save_foreign_1, run_id=scheduled__2025-09-02T23:46:00+00:00, execution_date=20250902T234600, start_date=20250903T145313, end_date=20250903T145315
[2025-09-03T14:53:15.735+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-03T14:53:15.774+0000] {taskinstance.py:3498} INFO - 4 downstream tasks scheduled from follow-on schedule check
[2025-09-03T14:53:15.776+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
